### PR TITLE
[SYCL] Add diagnostic on command group with multiple actions

### PR DESCRIPTION
**Pull request by romanovvlad**
Signed-off-by: Vlad Romanov <vlad.romanov@intel.com>

### CODE REVIEW

**Comment by vladimirlaz** [comment]
it is ok to have that check on run time for hard cases. But obvious cases may be detected and reported on compile time. Can we have that diagnostic in compiler?

**Comment by bader** [comment]
> it is ok to have that check on run time for hard cases. But obvious cases may be detected and reported on compile time. Can we have that diagnostic in compiler?

I don't see how this case can be detected at compile time. Do you have any ideas?

**Comment by vladimirlaz** [comment]
> > it is ok to have that check on run time for hard cases. But obvious cases may be detected and reported on compile time. Can we have that diagnostic in compiler?
> 
> I don't see how this case can be detected at compile time. Do you have any ideas?

I was thinking about counting functions call of command group lambda in SemaSYCL. But it looks like it fill cause false alarms in case of branching in the lamda

File: sycl/include/CL/sycl/handler.hpp [hunk 1]
```diff
@@ -190,6 +190,14 @@ class handler {
     return Storage;
   }
 
+  void throwIfActionIsCreated() {
+    if (detail::CG::NONE != MCGType)
+      throw sycl::runtime_error("Attempt to set multiple actions for the "
+                                "command group. Command group must consists of "
```
Comment (sergey-semenov) [inline sycl/include/CL/sycl/handler.hpp]:
```suggestion
                                "command group. Command group must consist of a"
```

File: sycl/test/basic_tests/diagnostics/handler.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// RUN: %clangxx -g -O0 -fsycl %s -o %t.out
```
Comment (sergey-semenov) [inline sycl/test/basic_tests/diagnostics/handler.cpp]: Any reason you need "-g -O0" flags?

Comment (romanovvlad) [reply sycl/test/basic_tests/diagnostics/handler.cpp]: Removed.

**Pull request merged**